hubblestack/utils/signing.py

```diff
@@ -63,39 +63,47 @@
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.serialization import load_pem_private_key
 
-MANIFEST_RE = re.compile(r'^\s*(?P<digest>[0-9a-fA-F]+)\s+(?P<fname>.+)$')
+MANIFEST_RE = re.compile(r"^\s*(?P<digest>[0-9a-fA-F]+)\s+(?P<fname>.+)$")
 log = logging.getLogger(__name__)
 
-def _our_byte_string(x, charmap='utf-8'):
+
+def _our_byte_string(x, charmap="utf-8"):  # pylint: disable=unused-argument
     # MONKEYPATCH OpenSSL._utils:byte_string to avoid "charmap" issues on POSIX
     # locale platforms
-    return x.encode('utf-8')
+    return x.encode("utf-8")
+
 
 OpenSSL._util.byte_string = _our_byte_string
 
 # "verification_log_timestamps" is a global dict that contains str path
 # and time() kv pairs. When the time() value exceeds the dampening_limit (3600 sec),
 # we reset time and set log level accordingly.
-verif_log_timestamps = {}
+VERIFY_LOG_TIMESTAMPS = {}
 # How often in seconds 3600 = 1 hour to set log level to log.error/critical
 # maybe set in /etc/hubble/hubble
-verif_log_dampener_lim = 3600
+VERIFY_LOG_DAMPENER_LIM = [3600]
```

Comment:
the only way i found to lose the `global` usage and have the value available in functions' inner scopes. i don't like it, myself, it can be changed, if needed.